waspc/src/Wasp/Generator/DbGenerator/Operations.hs

```diff
@@ -1,30 +1,37 @@
 module Wasp.Generator.DbGenerator.Operations
   ( migrateDevAndCopyToSource,
+    generatePrismaClient,
   )
 where
 
 import Control.Concurrent (Chan, newChan, readChan)
 import Control.Concurrent.Async (concurrently)
+import Control.Monad (when)
 import Control.Monad.Catch (catch)
 import qualified Path as P
 import StrongPath (Abs, Dir, Path')
 import qualified StrongPath as SP
+import System.Directory (doesFileExist)
 import System.Exit (ExitCode (..))
 import Wasp.Common (DbMigrationsDir)
 import Wasp.Generator.Common (ProjectRootDir)
-import Wasp.Generator.DbGenerator
+import Wasp.Generator.DbGenerator.Common
   ( dbMigrationsDirInDbRootDir,
     dbRootDirInProjectRootDir,
-    writeDbSchemaChecksumToFile,
+    dbSchemaChecksumFileInProjectRootDir,
+    dbSchemaFileInProjectRootDir,
   )
 import qualified Wasp.Generator.DbGenerator.Jobs as DbJobs
 import Wasp.Generator.FileDraft.WriteableMonad
   ( WriteableMonad (copyDirectoryRecursive),
   )
 import Wasp.Generator.Job (JobMessage)
 import qualified Wasp.Generator.Job as J
-import Wasp.Generator.Job.IO (printJobMessage)
+import Wasp.Generator.Job.IO (printJobMessage, readJobMessagesAndPrintThemPrefixed)
+import Wasp.Util (checksumFromFilePath, hexToString)
 
+-- | NOTE(shayne): This appears to be used instead of readJobMessagesAndPrintThemPrefixed during migration
```

Comment:
Yeah, I should move closer to the call instead 👍🏻

Reply:
I may actually nix it. Was thinking it may help in case something wants to refactor it and then doesn't see the messed up output but it may not require that special attention at all.